src/main/java/uk/gov/hmcts/befta/player/DefaultBackEndFunctionalTestScenarioPlayer.java

```diff
@@ -141,6 +143,7 @@ public void prepareARequestWithAppropriateValues() throws IOException {
 
     private void prepareARequestWithAppropriateValues(BackEndFunctionalTestScenarioContext scenarioContext)
             throws IOException {
+        runPrerequisitesSpecifiedInTheContext(scenarioContext);
```

Comment:
This should be the very last thing just before placing the call under test.

Reply:
moved outside of function as agreed